src/Workspaces/VisualBasic/Portable/Extensions/SyntaxNodeExtensions.vb

```diff
@@ -797,11 +797,51 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
         End Function
 
         <Extension>
-        Public Function IsLeftSideOfAnyAssignStatement(node As SyntaxNode) As Boolean
+        Public Function IsLeftSideOfSimpleAssignmentStatement(node As SyntaxNode) As Boolean
             Return node.IsParentKind(SyntaxKind.SimpleAssignmentStatement) AndAlso
                 DirectCast(node.Parent, AssignmentStatementSyntax).Left Is node
         End Function
 
+        <Extension>
+        Public Function IsLeftSideOfAnyAssignmentStatement(node As SyntaxNode) As Boolean
+            Return node IsNot Nothing AndAlso
+                node.Parent.IsAnyAssignmentStatement() AndAlso
+                DirectCast(node.Parent, AssignmentStatementSyntax).Left Is node
+        End Function
+
+        <Extension>
+        Public Function IsAnyAssignmentStatement(node As SyntaxNode) As Boolean
+            Return node IsNot Nothing AndAlso
+                SyntaxFacts.IsAssignmentStatement(node.Kind)
```

Comment:
i feel like this would just be `typeof node is assignmentstatementsyntax`, no?

Reply:
I matched the existing coding patterns used in this file and SyntaxNodeExtensions.cs

Reply:
works for me!